plenoptic/synthesize/eigendistortion.py

```diff
@@ -454,6 +459,6 @@ def plot_distorted_image(self,
 
         img_processed = process_image(image + alpha * dist)
         to_plot = torch.clamp(img_processed, 0, 1)
-        fig = po.imshow(to_plot, ax=ax, **kwargs)
+        fig = po.imshow(to_plot, ax=ax, plot_complex=plot_complex, **kwargs)
```

Comment:
this will fail right? you're no longer importing `po`.

Reply:
Oh -- you are still importing `po` in this script. Should change that, like you did in frontend.py

Reply:
done! ty.